### PR TITLE
Update electrs

### PR DESCRIPTION
**Pull request by pwltr**
Update electrs to remove "unhealthy" state

Contains https://github.com/runcitadel/docker-electrs/commit/8f13518d542e342e07dd244935d82c18f937b7a1

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -209,7 +209,7 @@ services:
         ipv4_address: $NEUTRINO_SWITCHER_IP
   electrum:
     container_name: electrum
-    image: ghcr.io/runcitadel/electrs:v0.9.5@sha256:5fdd76415645de14f31c43844dc143b1477f86872d2f73a041c5005d469ed510
+    image: ghcr.io/runcitadel/electrs:v0.9.6@sha256:cf08513b8aa6f081e42ea92c5903d520db2caedbc20aaccf70f844663bbc8646
```
Comment (AaronDewes) [inline docker-compose.yml:212]: Can you directly go to 0.9.7?

Comment (pwltr) [reply docker-compose.yml:212]: The build for that tag didn't go through, also I don't know if debian-bullseye and debian-11 isn't the same thing?

**Review by WilliamConnatser — APPROVED** [review]
This looks good to me, but I will be on the lookout if @AaronDewes wants to upgrade to `0.9.7`. :)

File: docker-compose.yml (hunk 1 above)
Comment (AaronDewes) [reply docker-compose.yml:212]: > The build for that tag didn't go through, also I don't know if debian-bullseye and debian-11 isn't the same thing?

It is, but Snyk somehow detects them as different 🤷‍♂️

**Pull request merged**